how_to_contribute_to_docs.md

```diff
@@ -0,0 +1,78 @@
+# How to Contribute to Docs
+
+## **Website Documentation Guidelines**
+If you haven't build website locally yet, run following commands inside `theupdateframework` directory (use your [fork](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://docs.github.com/articles/fork-a-repo&ved=2ahUKEwjexbXp6aqGAxXP1zgGHYMfB3AQFnoECAkQAQ&usg=AOvVaw3knkDqfZXHdtlZVtWiPqst) repository)
+```
+ $ yarn install
+ $ hugo server
+```
+This will run website locally on http://localhost:1313 by default. 
+
+The [content](https://github.com/theupdateframework/theupdateframework.io/tree/master/content) directory is used as a source for Markdown files (Docs), which Hugo uses to generate the TUF website resources.
+
+### **Add Menu on Website**
+___
+
+The starting step is to add a route to render your Markdown file.
+
+Navigate to the [config.toml](https://github.com/theupdateframework/theupdateframework.io/blob/master/config.toml) file.
+
+### **Create Menu with Submenu options**
+When Menu contains SubMenu options
+
+- Use the following format:
+```toml
+[[menu.main]]
+name = "<Menu>"
+identifier ="<Menu_Identifier>"
+weight = <order>
+```
+- Place the Menu at the relevent position in [config.toml](https://github.com/theupdateframework/theupdateframework.io/blob/master/config.toml) based on its weight.
+- Restart the Hugo server, to see changes.
+
+### **Create Menu without SubMenu options**
+
+When Menu does not contains any Submenu options.
+
+- Use the following format:
+```toml
+[[menu.main]]
+name = "<Menu>"
+url = "/<route>/"
+weight = <order>
+```
+
+- Place the section at the relevant position according to its weight in [config.toml](https://github.com/theupdateframework/theupdateframework.io/blob/master/config.toml).
+- Restart the Hugo server, to see changes.
+
+### **Add SubMenu options to Parent Menu**
+___
+
+- Use the following format:
+```toml
+[[menu.main]]
+name = "<SubMenu_Heading>"
+parent = "<Menu>"
+url = "/<route>"  #This should be in lowercase, relates to SubMenu heading, and should not conflict with other routes
+weight = <order>
+```
+
+- Position the SubMenu at the relevant place according to its weight in [config.toml](https://github.com/theupdateframework/theupdateframework.io/blob/master/config.toml).
+- Restart the Hugo server, to see changes.
+
+
+
+### **Markdown Files for Menu or SubMenu options**
+___
+
+- The Markdown file name should be `<route>.md` for consistency.
+- Front Matter is essential on top of every Markdown. In Hugo, the title is defined like this:
+```markdown
+---
+title: <title>
+---
+```
+else file content can be written according to [Markdown rules](https://www.markdownguide.org/basic-syntax/) and [Hugo Markdown Support](https://www.markdownguide.org/tools/hugo/#hugo-markdown-support). 
```

Comment:
Sorry, what are you trying to say here exactly? I think it's clear that files can be written in Markdown, but what does Hugo support mean here?

Reply:
@trishankatdatadog This isn’t common, but some Markdown processors allow you to do extra tasks, but Hugo doesn't support that. Please [see ](https://www.markdownguide.org/tools/hugo/#hugo-markdown-support), So i just try to highlight that here.